examples/angular/src/app/components/contentlets/contentlets.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, Input } from '@angular/core';
+import { Contentlet } from '@dotcms/client/src/lib/client/content/shared/types';
+import { GenericContentlet } from '../../utils';
+import { environment } from '../../../environments/environment';
+
+@Component({
+  selector: 'app-contentlets',
+  standalone: true,
+  imports: [],
+  template: `<ul class="flex flex-col gap-7">
+    @for(contentlet of contentlets; track contentlet.identifier) {
+    <li class="flex gap-7 min-h-16">
+      <a class="min-w-32 relative" [href]="contentlet.urlMap ?? contentlet.url">
+        <img
+          [src]="getImageUrl(contentlet)"
```

Comment:
@zJaaal As part of the example, should we encourage the customer to use the ngSrc directive?